lib/bolt/bolt_option_parser.rb

```diff
@@ -298,15 +311,22 @@ def initialize(options)
       update
     end
 
-    def hide_target_opts(toggle = true)
-      @nodes.hide = @query.hide = @rerun.hide = @targets.hide = toggle
+    def remove_excluded_opts
```

Comment:
Is this `include_opts` now? Despite it's verbosity I stand by my preference for the `include` over `exclude`